.github/workflows/build.yml

```
@@ -0,0 +1,86 @@
+name: Dolphin-AppImage
+
+on: 
+  push:
+
+jobs:
+  dolphin-appimage:
+    name: dolphin bundled as appimage
+    runs-on: ubuntu-18.04
+
+    steps:
+      # Checks out a copy of your repository on the ubuntu-latest machine
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # Runs the Dolphin action
+      - name: Prepare ccache timestamp
+        id: ccache_cache_timestamp
+        shell: cmake -P {0}
+        run: |
+          string(TIMESTAMP current_date "%Y-%m-%d-%H;%M;%S" UTC)
+          message("::set-output name=timestamp::${current_date}")
+      - name: cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.ccache
+          key: build-ccache-${{ steps.ccache_cache_timestamp.outputs.timestamp }}
+          restore-keys: |
+            build-ccache-
+      - name: Install Packages
+        continue-on-error: false
+        id: packages
+        run: |
+          sudo add-apt-repository -y ppa:beineri/opt-qt-5.15.2-bionic
+          sudo apt-get install -y build-essential dialog dpkg libbluetooth-dev liblz4-dev liblzma-dev libssl-dev libopus-dev libpng-dev libsystemd-dev \
+          libzip-dev libzstd-dev zlib1g-dev libasound2-dev libpulse-dev pulseaudio p7zip-full libsfml-dev libminiupnpc-dev libmbedtls-dev libpugixml-dev \
+          libbz2-dev liblzo2-dev qt515base qt5-qmake libxi-dev libavcodec-dev libudev-dev libusb-1.0-0-dev libevdev-dev libc6-dev libhidapi-dev libavformat-dev libavdevice-dev \
+          libfmt-dev libwayland-dev libxrandr-dev libglu1-mesa-dev libcurl4-openssl-dev x11-utils zenity wget curl git gettext ccache make cmake ninja-build \
+      - name: Build Dolphin
+        run: |
+          mkdir -p "$HOME/.ccache"
+          mkdir build
+          cd build
+          cmake .. -GNinja -DLINUX_LOCAL_DEV=true -DCMAKE_C_COMPILER=/usr/lib/ccache/gcc -DCMAKE_CXX_COMPILER=/usr/lib/ccache/g++
+          ninja
+      - name: Package Dolphin-Binary
+        run: |
+          mkdir -p $GITHUB_WORKSPACE/artifacts
+          cp -P build/Binaries/dolphin-emu $GITHUB_WORKSPACE/artifacts
+      - name: Package Dolphin-AppImage
+        env:
+          QT_BASE_DIR: /opt/qt515
+          QTDIR: $QT_BASE_DIR
+        run: |
+          export PATH=$QT_BASE_DIR/bin:$PATH
+          echo "PATH $PATH"
+          qmake -v
+          cd /tmp 
+          curl -sLO "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
```

Using only `s` suppresses errors.
```suggestion
          curl -sSLO "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
```